frontend/src/lib/URLParser.test.ts

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
-import { createHashHistory, createLocation } from 'history';
+import { createHashHistory } from 'history';
 import { URLParser } from './URLParser';
 
 const history = createHashHistory();
-const location = createLocation('/');
+const location = {
+  pathname: '',
+  search: '',
+} as any;
```

Comment:
`createLocation` is no longer a function from `history`, so I just used a based object instead.